Tasks/DotNetCoreCLI/task.json

```diff
@@ -140,7 +140,7 @@
             "name": "selectOrConfig",
             "type": "radio",
             "label": "Feeds to use",
-            "defaultValue": "config",
+            "defaultValue": "select",
```

Comment:
In order for this change to get picked up on deployment, we also need to bump the minor version number for the task in task.json (and package.json, but there is just for consistency)

Reply:
Great catch, thanks - just did so.

Reply:
Sorry, Alex, I misspoke.  Should have been the patch number, my bad!

Reply:
Fixed